### PR TITLE
fix(140): mention npm as a build tool

### PR DESCRIPTION
**Pull request by cRAN-cg**
improve definition of npm. Closes [#140 ]

### CODE REVIEW

File: intro.tex [hunk 1]
```diff
@@ -155,7 +155,7 @@ \section{Setting Up}\label{s:intro-setup}
 we will see in \chapref{s:pages} how to run JavaScript in a browser.
 
 \texttt{npm} is the Node \gref{g:package-manager}{Package Manager},
-a command-line tool for finding, installing, and updating JavaScript libraries.
+a command-line tool for finding, installing, updating, building and executing JavaScript libraries.
```
Comment (gvwilson) [inline intro.tex]: Oxford comma after "building" please ("a, b, and c" instead of "a, b and c")

File: keypoints/intro.tex [hunk 2]
```diff
@@ -6,5 +6,5 @@
 \item
   Node is a command-line interpreter for JavaScript, which can be used interactively or to run scripts in files.
 \item
-  NPM is the Node Package Manager, which can be used to find, install, and update libraries.
+  NPM is the Node Package Manager, which can be used to find, install, update, build and execute JavaScript libraries.
```
Comment (gvwilson) [inline keypoints/intro.tex]: comma after "build" please.

**Comment by cRAN-cg** [comment]
Oh apologies, I just liked the statement better like this. Will make appropriate changes.

**Comment by cRAN-cg** [comment]
> Thanks for the improvement - I can add the Oxford commas if you'd rather. May we please also add you to the acknowledgments, and if so, what name would you like us to use?

Chiranjeev Gupta will be fine. 🙂

Also I would really like to contribute more.

**Comment by cRAN-cg** [comment]
@gvwilson Could you please review.

**Pull request merged**